### PR TITLE
Fix errors in old Network API version

### PR DESCRIPTION
**Pull request by anton-evseev**
Fixed validation errors:
* OBJECT_MISSING_REQUIRED_PROPERTY_DEFINITION
  * Remove property `nextHopType` from `required` as there was no such property in definition
* EQUIVALENT_PATH
  * Put `get` and `put` under one path

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2016-03-30/network.json [hunk 1]
```diff
@@ -3282,45 +3282,6 @@
         "x-ms-long-running-operation": true
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/connections/{connectionSharedKeyName}/sharedkey": {
```
[the remark this answers is not in the document]
Comment (jianghaolu) [reply specification/network/resource-manager/Microsoft.Network/stable/2016-03-30/network.json]: I see! Then it's fine

**Comment by jianghaolu** [comment]
Sorry I missed your comment during the holidays. Please reopen and target `january-network-release` and assign to me. Thanks!

**Comment by MikhailTryakhov** [comment]
@jianghaolu it need to be merged ASAP. In master in this case. Please remove ```january-network-release``` - I create branches myself :)

**Comment by number213** [comment]
@jianghaolu @MikhailTryakhov rebased PR against `master`, please take a look

**Review by jianghaolu — APPROVED** [review]

**Comment by MikhailTryakhov** [comment]
@number213  please merge it to january branch as well

**Comment by nschonni** [comment]
Looks like this issue might be in the 2015-06-15 files as well https://travis-ci.org/Azure/azure-rest-api-specs/jobs/476680029#L690-L706

**Comment by number213** [comment]
Nick's right, I missed one API version. Opened PR for it: https://github.com/Azure/azure-rest-api-specs/pull/5002
Also, ported those changes to January branch: https://github.com/Azure/azure-rest-api-specs/pull/5003

**Pull request merged**